builder/base/kss_builder_base.js

```diff
@@ -1091,6 +1102,7 @@ class KssBuilderBase {
             section.example = templateRender(template, contextClone(exampleContext));
 
             section.modifiers.forEach(modifier => {
+              console.log(modifier);
```

Comment:
I suppose you forgot to remove this `console.log()` statement.

Reply:
^^ yes it's really Work In Progress with some debug code inside

Reply:
``` suggestion
```